config/Vocab.conf

```diff
@@ -833,6 +833,13 @@ neyðar-sím-kerfi hk
 Sunnefju-foss kk
 þrjú-leyti hk
 fjögur-leyti hk
+axlar-síður lo
```

Comment:
These additions to Vocab.conf slipped in by accident, cleaner if you could remove them, and the dbclean.py script. They'll be erged into the codebase via another PR.